frontend/src/utils/coins.ts

```diff
@@ -39,6 +39,9 @@ export interface CoinConfig {
 
 export type Denom = string
 
+export const USDC_DENOM =
+  "ibc/498A0751C798A0D9A389AA3691123DADA57DAA4FE165D5C75894505B876BA6E4"
```

Comment:
This appears to be the Osmosis denom:

https://github.com/cosmostation/chainlist/blob/4eb37d9be3975a82cf8820a7cf9ca3810a450e51/chain/osmosis/assets.json#L2908-L2927

The Neutron denom is available here:

https://github.com/cosmostation/chainlist/blob/4eb37d9be3975a82cf8820a7cf9ca3810a450e51/chain/neutron/assets.json#L395-L414

```
ibc/B559A80D62249C8AA07A380E2A2BEA6E5CA9A6F079C912C3A9E9B494105E4F81
```

Reply:
It's a simple enough change, I'll make that update.